spec.html

```diff
@@ -73,6 +55,7 @@
 <div id="metadata-block">
   <h1>About this Specification</h1>
   <p>The document at <a href="https://tc39.es/ecma262/">https://tc39.es/ecma262/</a> is the most accurate and up-to-date ECMAScript specification. It contains the content of the most recent yearly snapshot plus any <a href="https://github.com/tc39/proposals/blob/master/finished-proposals.md">finished proposals</a> (those that have reached Stage&nbsp;4 in the <a href="https://tc39.es/process-document/">proposal process</a> and thus are implemented in several implementations and will be in the next practical revision) since that snapshot was taken.</p>
+  <p>This document is available as <a href="">a single page</a> and as <a href="multipage/intro.html">multiple pages</a>.</p>
```

Comment:
it emits `index.html` right? this would be more future-proof.

```suggestion
  <p>This document is available as <a href="">a single page</a> and as <a href="multipage/">multiple pages</a>.</p>
```

Reply:
That adds an extra redirect. I'm happier hardcoding the place it redirects to.

Reply:
Can we make `intro.html` be `index.html` instead? It would lead to a better URL design: it's weird to have nothing (that doesn't redirect) at `/`, but something at `/intro.html`.

Reply:
> it's weird to have nothing (that doesn't redirect) at `/`, but something at `/intro.html`.

I'm having trouble parsing this; say more?

Reply:
I meant that the following:

```
/                    # introduction
/some-section
/some-other-section
```

…is IMHO a more intuitive URL design than:

```
/                   # redirects to /intro
/intro
/some-section
/some-other-section
```

Reply:
Ah, sure. @devsnek asked for that too. I don't have strong feelings, so I'll switch it to that at some point.

Reply:
OK, done.

Reply:
I did hardcode it as `index.html`, though, since the internal links point there rather than to `/` and it's nicer to only have one page in your history.

Reply:
imo long term, "index.html" shouldn't be hardcoded anywhere, including in the internal links.